jfrog_platform/CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# CHANGELOG - jfrog_platform
+All changes to the JFrog platform integration will be documented in this file.
+
+## [1.0.0]
+* Release of first Jfrog Platform integration with Artifactory logs
```

Comment:
Nit: usually the most recent version for our integrations is at the top of the file (see example https://github.com/DataDog/integrations-core/blob/master/airflow/CHANGELOG.md)

Reply:
done